### ISSUE TITLE
JSON Parsing Error

### ISSUE THREAD

**Issue by risphs** [ripe-org/mango#11]
After running the tests I have came across the error that it can't parse the data. This is probably due to the fact the int can't be stringified. 

```bash
Uncaught SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at WebSocket.eval (webpack-internal:///./node_modules/babel-loader/lib/index.js!./node_modules/@vue/cli-plugin-typescript/node_modules/ts-loader/index.js??clonedRuleSet-41.use[1]!./node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/App.vue?vue&type=script&lang=ts:43)
```

**Comment by risphs** [comment]
I have found the issue being with the WebSocket. This has been fixed in commit [3e3b066](https://github.com/ripe-org/mango-core/commit/3e3b066b07326b1648497a84ad1dc075854c9df0). Now we just have to fix the stringified data.

**Comment by risphs** [comment]
Fixed now we will just do tests before committing

**Comment by risphs** [comment]
Completely fixed in commit [31a4171](https://github.com/ripe-org/mango/commit/31a4171a5ae274edfe2544ff8994749d787342d5) and [3a0cd21](https://github.com/ripe-org/mango-core/commit/3a0cd213f3a2edf907c28038ea31a68e40c922b9). We are still running into issues with the outgoing headers named `CONSOLE-RUN` and `CURRENT-CPU-USAGE` however this will be fixed in a later commit/pull-request.

**Comment by risphs** [comment]
This will be unpinned
